docker-compose.yml

```
@@ -640,7 +640,9 @@ services:
   #     - SMARTHOST_PORT=587
   
   redis:
-    image: redis:6.2-alpine
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
```

```suggestion
      dockerfile: redis/Dockerfile
```

Maybe its better to keep all the redis related things in a folder called `redis/`

that way we would have `redis/redis.conf` and `redis/Dockerfile`